docs/admin_guide.md

```diff
@@ -908,7 +908,7 @@ With the following content:
 
 ```ini
 [program:messenger]
-command=php /var/www/mbin/bin/console messenger:consume async async_ap failed --time-limit=3600
+command=php /var/www/mbin/bin/console messenger:consume old async outbox deliver inbox resolve receive failed --time-limit=3600
 user=www-data
```

Comment:
Down below (or above) briefly explain the queues in a simple table, with DSN +  the meaning / usage of each queue. Let me help you:


```md
| Queue name | DSN        | Description |
|------------|------------|-------------|
| old        | RabbitMQ   | Some text.. |
| async      | RabbitMQ   |             |
| outbox     | RabbitMQ   |             |
| deliver    | RabbitMQ   |             |
| inbox      | RabbitMQ   |             |
| resolve    | RabbitMQ   |             |
| receive    | RabbitMQ   |             |
| failed     | PostgreSQL |             |
| dead       | PostgreSQL |             |
```

Reply:
I extended the documentation in the admin guide. Should be enough, shouldn't it?

Reply:
Yea that should work :) I did like my table tho hahahaha